### ISSUE TITLE
cannot build using Python 3.3, 3.4 and 3.5

### ISSUE THREAD

**Issue by shiodat** [jubatus/jubadump#39]
```
$ ./waf configure
Traceback (most recent call last):
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Scripting.py", line 97, in waf_entry_point
    run_commands()
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Scripting.py", line 149, in run_commands
    parse_options()
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Scripting.py", line 127, in parse_options
    Context.create_context('options').execute()
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Options.py", line 134, in execute
    super(OptionsContext,self).execute()
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Context.py", line 87, in execute
    self.recurse([os.path.dirname(g_module.root_path)])
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Context.py", line 128, in recurse
    user_function(self)
  File "/home/daats/jubatus/jubadump/wscript", line 6, in options
    opt.load('unittest_gtest')
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Context.py", line 81, in load
    module=load_tool(t,path)
  File "/home/daats/jubatus/jubadump/.waf3-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Context.py", line 316, in load_tool
    __import__(d)
TypeError: source code string cannot contain null bytes

```

**Comment by kmaehashi** [comment]
Same issue as https://github.com/jubatus/jubatus/pull/1025
We should replace `unittest_gtest.py` with https://github.com/tanakh/waf-unittest/pull/18

**Comment by shiodat** [comment]
Partially fixed via #42.

**Comment by kmaehashi** [comment]
Fixed via #43
